tests/glsl-intrinsic/subpass-input/subpass-input.slang

```diff
@@ -0,0 +1,69 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL): -target glsl -stage fragment -entry main -allow-glsl
```

Comment:
I don't think we want to test GLSL for subpass feature, do we?
The document says,
```
Subpass-input functions are only available when targeting a Vulkan fragment stage.
```

Reply:
`Subpass-input functions are only available when targeting a Vulkan fragment stage.` we do want to target subpass-input since Slang supports only Vulkan, so there is no downside to having a GLSL & SPIR-V test